src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionActions.tag

```diff
@@ -50,7 +50,7 @@
      data-toggle="tooltip"
      data-placement="top"
      style="display: inline-block; padding-right: 5px;">
-     <div class="btn-group margin-bottom-7px">
+    <div class="btn-group margin-bottom-7px">
```

Comment:
Your current changeset is correct. The one in master is [currently wrong](https://github.com/TEAMMATES/teammates/blob/d0b4dd17f22eb18d1f325201a440b32045aa6859/src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionActions.tag#L53) (5 spaces to the left of `div` instead of 4) due to #7312.